subprocess/subtree_oom.go

```diff
@@ -63,3 +68,58 @@ func getPidFromLog(line string) (int, bool) {
 	}
 	return pid, true
 }
+
+func ClearOOMHandler() http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		grip.Debug("clearing oom")
+		resp := NewOOMTracker()
+		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+		defer cancel()
+
+		if err := resp.Clear(ctx); err != nil {
+			grip.Error(err)
+			w.WriteHeader(http.StatusInternalServerError)
```

Comment:
I'd use the gimlet response writing format

Reply:
What do you mean?